tests/parsing/FilterTest.py

```diff
@@ -125,3 +125,11 @@ def test_filter_by_can_detect_null(self):
             # All bear plus 1 line holding the closing colour escape sequence.
             self.assertEqual(len(stdout.strip().splitlines()),
                              TEST_BEARS_COUNT + 1)
+
+    def check_by_style(self):
+        retval, stdout, stderr = execute_coala(
+            coala.main, 'coala', '-B' '--filter-by', 'can_detect')
+        self.assertEqual(retval, 0)
+        self.assertRaises(InvalidFilterException,
+                          InvalidFilterException,
+                          str(filter), FilterHelper.get_all_filters_str())
```

Comment:
could you use a custom string instead of invoking `str` on the python built-in function `filter`?
```
>>> str(filter)
"<class 'filter'>"
```

Reply:
@ayush1999  your commit message `changes made` doesn't follow the coala commit guidelines. Please refer to [this](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) and edit your commit message accordingly.